extensions/2.0/Vendor/EXT_feature_metadata/0.0.0/README.md

```diff
@@ -0,0 +1,665 @@
+# EXT\_feature\_metadata
+
+**Version 0.0.0** (DATE HERE)
+
+## Contributors
+
+* Peter Gagliardi, Cesium
+* Sean Lilley, Cesium
+* Bao Tran, Cesium
+* Sam Suhag, Cesium
+* Samuel Vargas, Cesium
+* Patrick Cozzi, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec. Depends on [`EXT_mesh_gpu_instancing`](https://github.com/KhronosGroup/glTF/pull/1691) for instanced features.
```

Comment:
>Depends on [`EXT_mesh_gpu_instancing`](https://github.com/KhronosGroup/glTF/pull/1691) for instanced features.

This can be removed until we decide to add it back